build/build_package.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Maybe move sh file `script` or `bin` folder and keep the dockerfiles in the root folder

Reply:
:+1: - build/bin/*?